tests/test_grid.py

```diff
@@ -620,6 +622,15 @@ def test_enforce_symmetry(self):
     @pytest.mark.unit
     def test_symmetry_volume_integral(self):
         """Test volume integral of a symmetric function."""
+        L = [3, 3, 5, 3]
+        M = [3, 6, 5, 7]
+        N = [2, 2, 2, 2]
+        NFP = [5, 3, 5, 3]
+        sym = np.array([True, True, False, False])
+        # to test code not tested on grids made with M=.
+        even_number = 4
```

Comment:
was this changed code or you just moved it?

Reply:
I think this just got moved